duneanaobj/StandardRecord/SRGArECAL.h

```diff
@@ -0,0 +1,34 @@
+////////////////////////////////////////////////////////////////////////
+// \file    SRGArECAL.h
+// \brief   Reconstructed GAr ECAL cluster object
+// \author  F. Martinez <f.martinezlopez@qmul.ac.uk>
+// \date    Jan. 2023
+////////////////////////////////////////////////////////////////////////
+
+#ifndef DUNEANAOBJ_SRGARECAL_H
+#define DUNEANAOBJ_SRGARECAL_H
+
+#include "duneanaobj/StandardRecord/SRVector3D.h"
+#include "duneanaobj/StandardRecord/SRParticleTruth.h"
+
+namespace caf
+{
+  class SRGArECAL
+  {
+    public:
+      SRVector3D position;  ///< ECAL cluster 3D position
+      float E = -999.;      ///< Energy in MeV corresponding to this cluster
+
+      int hits_in_cluster = -999; ///< Total number of ECAL hits in cluster
+
+      int ecal_id = -999;  ///< GArSoft ECALId for ECAL cluster
```

Comment:
These `_id` branches make me a tad nervous because without coming all the way back to the documentation here a user can easily mistake them for one of the many indices floating around (GArSoft, the SRGAr object, even a GEANT ID in truth info).  Some user macro way downstream is an easy place for someone to _think_ they know it means and make a mistake in practice...

I'd suggest `garsoft_ecal_id`, or if you have better ideas, by all means use them